tests/test_free_energy.py

```diff
@@ -156,20 +154,18 @@ def test_construct_differentiable_interface_fast():
     x_b = utils.get_romol_conf(st.mol_b)
     coords = st.combine_confs(x_a, x_b)
     box = np.eye(3) * 100
-    lam = 0.5
 
     potentials = vac_sys.get_U_fns()
     sys_params = [np.array(bp.params) for bp in potentials]
 
     for precision in [np.float32, np.float64]:
         U_ref = construct_differentiable_interface(potentials, precision)
         U = construct_differentiable_interface_fast(potentials, sys_params, precision)
-        args = (coords, sys_params, box, lam)
+        args = (coords, sys_params, box)
         np.testing.assert_array_equal(U(*args), U_ref(*args))
 
-        argnums = (0, 1, 3)
```

Comment:
Looks like this went from testing `[du_dx, du_dp, du_dl]` to `[du_dx]` since it no longer has `argnums = (0, 1)`. Is it intended to remove the `du_dp`?

Reply:
Good catch, this was accidental. Will fix.

Reply:
Fixed in f490d3c